app/assets/stylesheets/components/_alert.scss

```diff
@@ -4,7 +4,6 @@
   background-color: #d0e9c6;
   font-size: 0.8rem;
   padding: 0.5em;
-  @include span-columns(10);
```

Comment:
I had the same thought, and when I looked it seemed like those are rendered with the class `flash-alert`:

```erb
<div class="flashes">
  <% user_facing_flashes.each do |key, value| -%>
    <div class="flash-<%= key %>"><%= value %></div>
  <% end -%>
</div>
```

Ref: https://github.com/18F/dolores-landingham-slack-bot/blob/c88f124da52a741db47c66e4e44d529eab36862c/app/views/application/_flashes.html.erb#L4

The styles for that seems to live in `app/assets/stylesheets/components/_flashes.scss`.

Ref: https://github.com/18F/dolores-landingham-slack-bot/blob/c88f124da52a741db47c66e4e44d529eab36862c/app/assets/stylesheets/components/_flashes.scss